core/src/components/textarea/textarea.scss

```diff
@@ -74,6 +72,8 @@
   flex: 1;
 
   background: var(--background);
+
+  white-space: pre-wrap;
```

Comment:
This change is necessary because the old code was causing whitespace to appear in the textarea if you added newlines when slotted in a label:

```html
<ion-textarea label-placement="start" value="hi@ionic.io">
  <div slot="label">Email <span class="required">*</span></div>
</ion-textarea>
```

This code was added as a result of https://github.com/ionic-team/ionic-framework/issues/18495 to make it so you can set `white-space` on `ion-textarea` and have it inherit to the native textarea. This is not a pattern we should recommend using the modern form controls as it will impact both the textarea and the label, which is not always what developers want. Instead we should encourage developers to target the label or the textarea directly when changing white space.

Reply:
You'll also see a few screenshot diffs because this is impacting textareas in `main`: https://github.com/ionic-team/ionic-framework/pull/27647/commits/cf85fa4a4164d96d3408cdb7f27488ff5d773cf2. The buggy behavior is not common right now but it will likely become more common once label slots become available.

Reply:
This makes sense to me.